debian/patches/0060-add-simd-optimized-tonemapx-filter.patch

```diff
@@ -2937,12 +2943,14 @@ Index: FFmpeg/libavfilter/vf_tonemapx.c
 +        return AVERROR(ENOMEM);
 +    if (!s->delin_lut && !(s->delin_lut = av_calloc(32768, sizeof(uint16_t))))
 +        return AVERROR(ENOMEM);
++    if (s->lut_delin_max <= 0)
++        return AVERROR(EINVAL);
 +
 +    for (i = 0; i < 32768; i++) {
-+        double v1 = (i - 2048.0f) / 28672.0f;
++        double v1 = (i - s->lut_off) / 32767.0f;
```

Comment:
Why it is not divided by limited range `28672.0f` when the input is in limited range now? In my implementation this is changed with the `lut_off`.

Reply:
Because doing so gives incorrect colors based on testing, too bright and loses too much detail.

![off](https://github.com/user-attachments/assets/bdd9bf86-3f4a-4304-9fba-2eaf11c01411)
![fine](https://github.com/user-attachments/assets/1e9a9879-0828-4750-848d-7ea2f1563a0d)

Reply:
This is due to still incorrect range handling though, as the output rgb color after yuv2rgb is always in full range this limited range lut should not be used on those colors. I'm going to push my version soon.